crates/apub/assets/gnusocial/objects/note.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
Did you mean to delete all these ones that don't have anything to do with friendica?

Reply:
Yes, check the second commit. Basically, these objects are already being tested as part of `Create/Note` and similar activity tests, so there is no real reason to add them twice. Except maybe to figure out what exactly isnt working, so maybe better to keep them. Now i'm not sure myself.

Reply:
IMO it would make sense to keep them, if only for the reason that they individually might add / remove things in the future, and these separate files are a place to keep track of that.